test/client_test.js

```diff
@@ -18,14 +18,32 @@ describe('Client', () => {
     expect(this.fhirClient.baseUrl).to.deep.equal(new URL(baseUrl));
   });
 
+  describe('invalid instance', () => {
```

Comment:
I believe `instance` in these tests refers to an instance of `Client`, so it isn't accurate to say that this is an invalid client instance. I think it would make sense to structure the tests as follows:
```
describe 'Client'
  it 'initializes with config' (or nest this inside of a describe new?)
  describe '#smartAuthMetadata'
    context 'when SMART URIs are present'
      it 'does successful stuff'
    context 'when SMART URIs are not present'
      it 'does unsuccessful stuff'
  describe '#read'
    etc.
```